packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.scss

```diff
@@ -35,7 +35,6 @@
 
     .gux-left,
     .gux-right {
-      color: var(--gse-ui-calendarMenu-month-monthHeader-foregroundColor);
```

Comment:
is the arrow color wrong here? ![image](https://github.com/user-attachments/assets/20dc5439-cb06-4593-8a74-529ce4d37bd6)

Reply:
That is a surprise to me. `--gse-ui-calendarMenu-month-monthHeader-foregroundColor` is undefined so I assumed `color: var(--gse-ui-calendarMenu-month-monthHeader-foregroundColor);` was the same as having no color but it is not.
Have added it back in for now and will fix in a specific PR related to this type of case.